website/docs/concepts/function-components/properties.mdx

```diff
@@ -262,3 +262,19 @@ fn App() -> Html {
     html! {<HelloWorld ..pre_made_props />}
 }
 ```
+
+## Anti Patterns
+
+While almost any Rust type can be passed as properties, there are some anti-patterns that should be avoided.
+These include, but are not limited to:
+
+1. Using `String` type instead of `AttrValue`
+   **Why is this bad?** `String` can be expensive to clone.
+   Cloning is often needed when the prop value with hooks and such. `AttrValue` is either
+   a reference-counted string (`Rc<str>`) or a `&'static str`, thus very cheap to clone.
```

Comment:
Would this be a good place to plug the whole of `[implicit-clone](https://crates.io/crates/implicit-clone)`?

Reply:
I added a note section with the link to that crate